.github/scripts/lib/frame-credit-ledger-core.ps1

```
@@ -2726,6 +2773,416 @@ function Invoke-CreditInputHarvest {
+
+        return $v
+    }
+
+    # -----------------------------------------------------------------------
+    # Helper: render a single credit hashtable as YAML list entry lines.
+    # Known scalar fields that survive the reader (round-trip coverage):
+    #   port, adapter, status, run_index, evidence, terminal-step-id
+    # Nested list fields (judge-score.findings[], prosecution-passes,
+    # files-checked) are out of scope for this builder.
+    # -----------------------------------------------------------------------
+    function script:Render-CreditEntry {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Render-CreditEntry' uses an unapproved verb.

(PSUseApprovedVerbs)